opm/autodiff/BlackoilModelEbos.hpp

```diff
@@ -456,7 +456,10 @@ namespace Opm {
 
             for (int cell_idx = 0; cell_idx < nc; ++cell_idx) {
                 double dp = dx[cell_idx][flowPhaseToEbosCompIdx(0)];
-                reservoir_state.pressure()[cell_idx] -= dp;
+                //reservoir_state.pressure()[cell_idx] -= dp;
+                double& p = reservoir_state.pressure()[cell_idx];
+                p -= dp;
+                p = std::max(p, 1e5);
```

Comment:
I usually limit the pressure change per iteration to something like 20% of the absolute value, i.e., 

```c++
if (p > 0)
    dp = std::max(dp, p*0.2);
else
    dp = std::max(dp, -p*0.2);
```

(but I don't know if this works better...)